.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           cd build
           [ -d xt ] && prove --timer --lib --recurse --jobs $(nproc) --shuffle xt
-          HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
+          HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/,+ignore,^/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
```

Comment:
Perhaps this should be expressed as:

```suggest
HARNESS_PERL_SWITCHES=-MDevel::Cover=+ignore,^t/,^/ prove --timer --lib --recurse --jobs $(nproc) --shuffle t/
```

Reply:
Yes, that'd be better. However, I think the problem came from an entirely different angle than I originally expected. (Thanks to the people in #perl on OFTC for taking a peek at our code base!) So, I have a completely new proposal now.